charts/monasca/values.yaml

```diff
@@ -169,6 +207,13 @@ kafka:
   service:
     type: ClusterIP
     port: 9092
+  resources:
+    requests:
+      memory: 2Gi
```

Comment:
You could drop these to 1Gi and 2Gi if you needed to use less resources. I think